resources/lang/en/numeric.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'mb' => 'MB',
```

Comment:
move it to generic, no use having a `numeric` lang file that only contains `mb` (filename made me think of other values than "MB" in here too)

Reply:
Yeah, fair point, shouldn't be more than that, was just to cater about the weird french translations (MB => MO) but yeah, gonna move it to generic.